data/migrations/0005_pets.js

```diff
@@ -0,0 +1,22 @@
+exports.up = async (knex) => {
+  await knex.schema.createTable('pets', (table) => {
+    table.increments('id');
+    // table.bigint('pet_types_id').references('id').inTable('pet_types');
```

Comment:
Since the pet_types table is completely removed from the schema this commented out code can be removed.